proto/zetachain/zetacore/crosschain/cross_chain_tx.proto

```diff
@@ -27,6 +27,12 @@ enum TxFinalizationStatus {
   Executed = 2;     // the corresponding tx is executed
 }
 
+enum ConfirmationMode {
+  option (gogoproto.goproto_enum_stringer) = true;
+  SAFE = 0; // an inbound/outbound is confirmed using safe confirmation count
+  FAST = 1; // an inbound/outbound is confirmed using fast confirmation count
+}
+
```

Comment:
Do we need `ConfirmationMode` in Inbound and Outbound params separately? 
If not would you consider moving this to a higher level to put under the main struct directly

Reply:
I was thinking about a scenario when a depositAndCall is confirmed on `SAFE` mode while a corresponding revert outbound is confirmed on `FAST` mode. wdyt?

Reply:
If we want to support something like that, we would need separate confirmation counts. But if we do not need to, it would be simpler just to have one confirmation count for the entire cctx

Reply:
The thing is that, the `FAST` is an opt-in mode (can be disabled any time, or not applied due to rate limit) and there is no guarantee it's always same in whole CCTX. So, a `FAST` observed inbound, could always have the chance to be observed using a `SAFE` mode for its revert outbound.

Reply:
we can close this one and leave all discussions here https://github.com/zeta-chain/node/pull/3506#discussion_r1949941322